hoomd/md/pytest/test_methods.py

```diff
@@ -274,6 +274,33 @@ def test_npt_attributes_attached_3d(simulation_factory,
     assert npt.gamma == 2.0
 
 
+def test_npt_thermalize_extra_dof(simulation_factory,
+                                  two_particle_snapshot_factory):
+    """Tests that NPT.thermalize_extra_dof can be called."""
+    all_ = hoomd.filter.All()
+    constant_t = hoomd.variant.Constant(2.0)
+    constant_s = [
+        hoomd.variant.Constant(1.0),
+        hoomd.variant.Constant(2.0),
+        hoomd.variant.Constant(3.0),
+        hoomd.variant.Constant(0.125),
+        hoomd.variant.Constant(.25),
+        hoomd.variant.Constant(.5),
+    ]
```

Comment:
These should work without the explicit conversion to a constant variant right?

Reply:
Yes. This was copied and pasted from a test where the explicit object was needed for `is` testing.